integrations/aws-ecs-monitoring.mdx

```diff
@@ -1,11 +1,22 @@
 ---
-title: "AWS ECS Monitoring"
+title: "Amazon ECS"
 ---
 
-## ECS + Fargate setup Monitoring
+# Overview
 
-For Each ECS Task that you want to monitor, add this one sidecar container in your task definition.
-```
+Amazon Elastic Container Service (Amazon ECS) is a fully managed container orchestration service that helps you easily deploy, manage, and scale containerized applications. With the Middleware Agent (MW Agent), you can effectively monitor your ECS containers and tasks running on ECS EC2 instances and ECS Fargate in your cloud, on-prem, and hybrid architectures.
+
+<Note> The Middleware Agent does not currently support the Amazon ECS Anywhere approach </Note>
+
+# Fargate Setup
+
+Use the Middleware and AWS Fargate integration to monitor your applications without having to manage servers. Run containers wihtout having to manage servers or clusters of Amazon EC2 instances.
```

Comment:
These two sentences sound the same. Maybe an accident to keep both? I think we're good with just the first one.